DESCRIPTION

```diff
@@ -94,7 +94,8 @@ Imports:
 Suggests:
     testthat,
     knitr,
-    vegan
+    vegan,
+    rmarkdown
```

Comment:
let's stick with just `knitr`, so remove this `rmarkdown` dep

Reply:
remove rmarkdown